README.txt

```diff
@@ -0,0 +1,46 @@
+### Specifications for the WetlandElevationChangeTable.csv ###
+
+FileName
+########
+Type - Text
+Null - False, cannot contain null values
+Desc - Field contains the names of each input point feature
+		class or shapefile. Names must exactly match the file
+		name, and will fail if aliases are supplied. Do not supply the file extension. One row represents one input file. 
```

Comment:
Could use an example, showing what the filenames are underneath and what you actually should supply here.
Some users may be confused about why they don't have to supply any directory information.